### ISSUE TITLE
jaxws-maven-plugin not working for Java 11 on Windows

### ISSUE THREAD

**Issue by amarchino** [eclipse-ee4j/metro-jax-ws#135]
Greetings.

When I try to execute the goal `wsimport` from the jaxws-maven-plugin with Java 11, the plugin generates the sources in `${project.basedir}/target/generated-sources/wsimport`, yet it fails to move them to the requested source folder.

The error given is
```
org.apache.maven.plugin.MojoExecutionException: Mojo failed - check output
  at org.jvnet.jax_ws_commons.jaxws.AbstractJaxwsMojo.exec (AbstractJaxwsMojo.java:367)
  [...]
```
yet no other output is given.

I may add the following, for debug purposes:
- the error is found on Windows OS
- I am using java 11 (openjdk version "11.0.7" 2020-04-14)
- I am using the latest version of the plugin 3.0.0-M2
- the classes found in `${project.basedir}/target/generated-sources/wsimport`, when moved to the source folder, actually work correctly

Any idea of what the problem may be?
Thanks!

**Comment by smillidge** [comment]
Not sure about the error but I wouldn't use 3.0.0-M2 version unless you are using the jakarta namespace rather than the javax namespace. That milestone has been created as part of the in-progress Jakarta EE 9 release.

**Comment by amarchino** [comment]
True.
Yet the same problem is present with the latest 2.x version (which is 2.3.3 if I'm not mistaken).
I thought that maybe the problem was resolved on the newest versions, and so I tested against those; yet also with the previous version the problem still exists.

Also, with the old version the sources seems not to be copied to the given `<destDir>`, but if at all that would be a different issue (so as not to pollute this one with extraneous problems that may not be linked to the original one).

Sorry for having reported incompletely!

**Comment by smillidge** [comment]
We'd welcome a PR if you want to investigate?

**Comment by lukasj** [comment]
can you be more specific? How is your project different from [test projects](https://github.com/eclipse-ee4j/metro-jax-ws/tree/master/jaxws-ri/extras/jaxws-maven-plugin/src/test/it) and/or [samples](https://github.com/eclipse-ee4j/metro-jax-ws/tree/master/jaxws-ri/extras/jaxws-maven-plugin/src/it) being used?

...maybe you could attach a simple pom showing the issue you have

**Comment by amarchino** [comment]
I tried to make some tests both against one of the test projects ([wsimport22](https://github.com/eclipse-ee4j/metro-jax-ws/tree/master/jaxws-ri/extras/jaxws-maven-plugin/src/test/it/wsimport22)) and a simplified extract of the project I am working on (published on Github [here](https://github.com/amarchino/test-jaxws-maven-plugin)).
Please note that the sample project is stripped to the bone in terms of pom.xml and code, but contains one of the WSDL I'm actually trying to parse, alberit with a "sanified" namespace and endpoint.

The results are as follow:
- Linux distro (Ubuntu)
  - the code from the test project works correctly
  - the code from the sample progect works correctly
- Maven on Docker (because why not?)
  - the code from the test project works correctly
  - the code from the sample progect works correctly
- Windows
  - the code from the test project does not work correctly
  - the code from the sample progect does not work correctly

Stranger still: on some Windows machines both the sample project and the test project work flawlessly (as do my main project), on others it does not.
I have prepared a log of the execution (for the sample project) in case it may shine some light on the problem... [jaxws-maven-plugin.error.maven.log](https://github.com/eclipse-ee4j/metro-jax-ws/files/4618357/jaxws-maven-plugin.error.maven.log)

Ah, one last thing: the error seems to be thrown after the execution of wsimport from the command line (line 331 og the attached log, or class [AbstractJaxwsMojo](https://github.com/eclipse-ee4j/metro-jax-ws/blob/2.3.2-RELEASE/jaxws-ri/jaxws-maven-plugin/src/main/java/org/jvnet/jax_ws_commons/jaxws/AbstractJaxwsMojo.java#L367)).
May this therefore be a problem of wsimport and not of the wrapping Maven plugin?

**Comment by lukasj** [comment]
thanks for the log! Looking at:
```
[DEBUG] cmd.exe /X /C "D:\Programs\Java\jdk-11.0.6+10\bin\java.exe
-cp /D:/maven/repository/com/sun... org.jvnet.jax_ws_commons.jaxws.Invoker
com.sun.tools.ws.wscompile.WsimportTool
-pathfile C:\Users\INTERL~1\AppData\Local\Temp\jax-ws-mvn-plugin-cp13252341928933258167.txt
-keep..."
File not found - .cmd_aliases
```

following two things look  suspicious:
* `-cp /D:/maven/` - the leading `/` should probably not be there
* `File not found - .cmd_aliases` - not sure what exactly this means nor if it can be related to the leading `/`

**Comment by lukasj** [comment]
@amarchino I tried something... Can you try 3.0.0-SNAPSHOT in few hours and let me know, please? To set up snapshot repo in your project, follow [the docs](https://eclipse-ee4j.github.io/metro-jax-ws/jaxws-maven-plugin/examples/using-snapshots.html) (wording has been fixed in #141 )

**Comment by amarchino** [comment]
I tried the 3.0.0-SNAPSHOT, but it still failed at first. Or, well... 3.0.0-SNAPSHOT for the jaxws-maven-plugin dependency, and 3.0.0-M2 for jaxws-ri, since the snapshot for this second project is still not out.

I tried to do away with the error `File not found - .cmd_aliases`, which was just a way to emulate the auto-loading of alias features of Linux on Windows (cfr. [this post](https://stackoverflow.com/a/21040825/2850367) on Stack Overflow)... And it magically started working.

I must admit that I am not that knowledgeable about how the Windows Command Processor works, but it seems that if any of the commands executed return an error, the entire command prompt returns an errror (whereas on Linux the exit code is that of the last executed command).

After having tried this, I tried the execution with the 2.3.3 version of both plugin and runtime, and it worked correctly!

Thanks for your work!! And I am very sorry for uselessly spending your time on a machine configuration problem.
